soccer/StripChart.cpp

```diff
@@ -47,6 +49,30 @@ void StripChart::function(Chart::Function* function) {
     }
 }
 
+void StripChart::xport() {
+    float newMin = _minValue;
+    float newMax = _maxValue;
+    for (unsigned int x = 0; x < _functions.size(); x++) {
+        auto function = _functions[x];
+        
+        std::ofstream outfile("chartData.csv");
+        outfile << "Time,Value\n";
+
+        for (unsigned int i = 0; i < _history->size(); ++i) {
+            float v = 0;
+            if (_history->at(i) && function->value(*_history->at(i).get(), v)) {
+                if (autoRange) {
```

Comment:
Is the stuff in this if statement needed? You don't seem to be using newMin or newMax anywhere?